paradedb/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+TRIES=3
+export PGPASSWORD='mypassword'
+
+cat queries.sql | while read query; do
+    sync
+    echo 3 | sudo tee /proc/sys/vm/drop_caches
+
+    echo "$query";
+    for i in $(seq 1 $TRIES); do
+    psql -h localhost -U myuser -d mydb -p 5432 -t -c "CALL paradedb.init();" -c '\timing' -c "$query" | grep 'Time'
```

Comment:
`init()` must be called one per connection. Because each ClickBench query runs in a new connection, we call it once before each to avoid modifying the ClickBench framework